backend/frontend/src/screens/BookingInfoScreen.js

```diff
@@ -391,14 +393,13 @@ function BookingInfoScreen() {
                         <span className="text-white dark-yellow-bg color-white me-2 d-flex justify-content-center align-items-center">
                           4.5
                         </span>
-                        <span>300 Reviews</span>
+                        <span>300 {t("reviews")}</span>
```

Comment:
I think this has to be dynamic, instead of concatenation of 300 and Reviews and same for below organizationDescription. This applies to all the places wherever you need to add dynamic values.

Reply:
Sure, I will change it accordingly to wherever it's needed.